ext/storage_sources/s3_store/s3_store.cxx

```diff
@@ -28,23 +28,30 @@
 
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
+#include <sys/stat.h>
+#include <fstream>
+#include <errno.h>
 
 #include <aws/core/Aws.h>
 #include "aws_bucket_conn.h"
 
 #define UNUSED(x) (void)(x)
-
+#define FS2S3(fs) (((S3_FILE_SYSTEM *)(fs))->s3_storage)
 /* S3 storage source structure. */
 typedef struct {
     WT_STORAGE_SOURCE storage_source; /* Must come first */
     WT_EXTENSION_API *wt_api;         /* Extension API */
+    uint64_t op_count;
```

Comment:
Let us have a separate counter for the object exists case. I imagine each S3 request will have a counter of its own. We can put that as a counter struct inside S3StorageSource.